api/edgeproto/appinst.proto

```diff
@@ -283,6 +283,8 @@ message InstPort {
   bool internal_vis_only = 10;
   // Port ID for NBI compatibility
   string id = 11;
+  // Service name for Kubernetes port, to distinguish if there are multiple of the same port on the App
```

Comment:
I changed it to `Service name for Kubernetes port, use with a custom manifest or Helm chart that uses same port number on different services in the app`. It's for Apps with multiple containers (i.e. a user-supplied manifest, or helm chart), where there may be multiple services using the same port. For example maybe it's running a DB and a webserver that are both use port 443. This is typically find because the ingress can route to either one based on hostname. This scenario isn't fully supported right now but we may run into it at some point.

To clarify further, the main issue here is that I believe we should be responsible for deploying the ingress objects, because we are setting the hostname and issuing the TLS certs. But then we need to be able to figure out, for each HTTP port on the App, which kubernetes service+port it routes to. If it's a custom manifest or Helm chart, we need to search the possible services and match the App port to the service+port. This annotation is needed if we can't figure that out just by looking at port numbers.